frontend/simple/js/state.js

```diff
@@ -135,6 +188,7 @@ const actions = {
       }
       // TODO: verify each entry is signed by a group member
       await db.addLogEntry(contractId, entry)
+      entry.data.hash = hash
```

Comment:
The hash is not on the data. How can that be? The data is what's hashed. That's impossible. The hash is a level above, on `entry.hash`.